routes/api/games.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const passport = require('passport');
+const Game = require('../../models/Game');
+// const getFirstWord method required to fetch first word from database
+// getNextWord
+
+// Frontend sends userId in post request
+router.post('/game', (req, res) => {
+  //is this the route we want?
+  const userId = req.userId;
```

Comment:
I think we'll want to decode/authenticate the JWT sent with the request, but this will work for testing. Maybe we can pull the authentication out into a separate utils file and then just do a one-liner here.